### PR TITLE
Update of API docs with more details for example payloads and better …

### PR DESCRIPTION
**Pull request by mswiderski**
…descriptions

@sterobin @cristianonicolai here are the updated docs and swagger files

### CODE REVIEW

**Comment by sterobin** [comment]
@mswiderski @cristianonicolai, I see "Controller API" was changed to "Business Central API". To be clear, this API should render as "Business Central API" when a user uses Central as the controller, but should render as "Controller API" when a user uses the headless controller, without Central. Is this still the case, even with the change in this PR? Just making sure.

**Comment by cristianonicolai** [comment]
@sterobin it depends from where you get the swagger.json file. I guess in this case, @mswiderski used BC when I used to run the standalone controller to generate the file. In the usage it still the same, the name will be changed according to where you run it. In here I guess we've to decide what makes more sense to use. Because we named the section as _Controller REST API_, I would prefer to keep that consistent and not show as BC.

**Comment by sterobin** [comment]
I agree with leaving it as “Controller REST API”.

**Comment by mswiderski** [comment]
So what should I do here? Or maybe Cristiano you have the standalone running and ca generate the swagger for it?

**Comment by cristianonicolai** [comment]
@mswiderski sure, just remove the controller related updates from your PR so I can generate and do a new PR

**Comment by mswiderski** [comment]
Since you have commit rights you can update this PR directly

**Comment by cristianonicolai** [comment]
@mswiderski done

**Comment by cristianonicolai** [comment]
@sterobin can you check this again, thanks.

File: api-docs/controller/paths.adoc [hunk 1]
```diff
@@ -862,15 +862,17 @@ POST /controller/management/servers/{serverTemplateId}/containers/{containerId}
...
-|Type|Name|Description|Schema
+|Type|Name|Description|Schema|Default
 |**Path**|**containerId** +
-__required__|ID of the KIE container to be updated|string
+__required__|ID of the KIE container to be updated|string|
 |**Path**|**serverTemplateId** +
-__required__|ID of the KIE Server template associated with the KIE container|string
+__required__|ID of the KIE Server template associated with the KIE container|string|
+|**Query**|**resetBeforeUpdate** +
+__optional__|Allows to reset the current environment aborting active process instances before updating when the server runs on development mode. Optional, defaults to false|boolean|`"false"`
```
Comment (sterobin) [inline api-docs/controller/paths.adoc]: Suggest this for consistency:

``
Determines whether active processes are aborted (reset) before updating when the server runs in development mode (default: `false`)
```

File: api-docs/controller/swagger.json [hunk 2]
```diff
@@ -207,6 +207,13 @@
...
           }
+        }, {
+          "name" : "resetBeforeUpdate",
+          "in" : "query",
+          "description" : "Allows to reset the current environment aborting active process instances before updating when the server runs on development mode. Optional, defaults to false",
```
Comment (sterobin) [inline api-docs/controller/swagger.json]: Suggest my description from above.

File: api-docs/controller/paths.adoc (hunk 1 above)
Comment (cristianonicolai) [reply api-docs/controller/paths.adoc]: @sterobin I created another PR to handle this update, see https://github.com/kiegroup/droolsjbpm-integration/pull/1778
If you don't mind, I prefer to merge this PR as is and get that change in here using a separated PR.

Comment (sterobin) [reply api-docs/controller/paths.adoc]: Okay, go ahead and merge then.

**Pull request merged**